helm/0.1.0/README.md

```diff
@@ -0,0 +1,225 @@
+# Helm Job-Executor Integration
+
+This integration shows you how to leverage [Job-executor-service](https://github.com/keptn-contrib/job-executor-service) for deployment with `helm` on your Kubernetes cluster.
+
+**Please note, that the instructions provided here serve the purpose of enabling you to run and configure your Kubernetes deployments using job-executor-service. They are in no way intended to be a complete reference of Kubernetes, nor Helm, nor Keptn.**
+
+## Installation and Configuration
+
+### Step 1: Install the Job-Executor in your cluster
+
+Install [Job-Executor documentation](https://github.com/keptn-contrib/job-executor-service), and make sure it is subscribed to the following Keptn Cloud Events:
+
+* `sh.keptn.event.deployment.triggered` (for actual deployments)
+* `sh.keptn.event.rollback.triggered` (for rollback sequence, i.e., when a previous deployment does not go well)
+* `sh.keptn.event.action.triggered` (for remediation sequence)
+
+**Example Installation Instruction for Job Executor**
+```bash
+TASK_SUBSCRIPTION='sh.keptn.event.deployment.triggered\,sh.keptn.event.rollback.triggered\,sh.keptn.event.action.triggered'
+JES_VERSION=0.2.5-next.0
+JES_NAMESPACE=keptn-jes
+
+helm upgrade --install --create-namespace -n ${JES_NAMESPACE} \
+  job-executor-service https://github.com/keptn-contrib/job-executor-service/releases/download/${JES_VERSION}/job-executor-service-${JES_VERSION}.tgz \
+  --set remoteControlPlane.autoDetect.enabled="true",remoteControlPlane.topicSubscription="${TASK_SUBSCRIPTION}",remoteControlPlane.api.token="",remoteControlPlane.api.hostname="",remoteControlPlane.api.protocol=""
+```
+
+**In case you have installed helm-service, please uninstall it**
```

Comment:
```suggestion
**If you have installed helm-service, please uninstall it**
```
Should this be done before or after you  install JES?

Reply:
Actually doesn't matter.